packages/lib/services/membership.ts

```diff
@@ -80,9 +80,9 @@ export const updateMembership = async (
   } catch (error) {
     if (error instanceof Prisma.PrismaClientKnownRequestError && error.code === "P2016") {
       throw new ResourceNotFoundError("Membership", `userId: ${userId}, teamId: ${teamId}`);
-    } else {
-      throw error; // Re-throw any other errors
     }
+
+    throw error;
```

Comment:
@rotimi-best should we maybe also here throw a new UnknownError with the message of the original error? This way we only through our own errors inside the services. What do you think?

Reply:
Updated, please let me know what you think.